src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -547,7 +547,7 @@ void EnGinkoMan_BankAward2(EnGinkoMan* this, GlobalContext* globalCtx) {
         } else {
             Actor_ChangeAnimation(&this->skelAnime, animations, GINKO_SITTING);
             func_801518B0(globalCtx, 0x47B, &this->actor);
-            this->curTextId = 0x47B; // "Is that so?  Think it over, little guy!  So what are you gonna do?"
+            this->curTextId = 0x47B; // "That's what they call interest!"
```

Comment:
Is this the actual quote of the text Id? This wasn't you, but we've been trying to move away from having actual quotes inside the repo, and if comments help, to have a summary of what the text is saying instead. And then leaving actual text as part of asset extraction. 

So this comment could either be removed or summarized. Up to you :p

Reply:
Elliptic said the other day that text comments *should* be added...

Reply:
My understanding is that comments *should* be added, but direct quotes from the game (unless it's unavoidable in the case where the text is just something like "huh?") should *not* be added.